rest/config.go

```diff
@@ -711,6 +711,13 @@ func (dbConfig *DbConfig) Redacted() (*DbConfig, error) {
 		return nil, err
 	}
 
+	err = config.redactInPlace()
+	return &config, err
+}
+
+// redactInPlace modifies the given config to redact the fields inside it.
+func (config *DbConfig) redactInPlace() error {
+
```

Comment:
Not strictly related to this PR, but should we only be redacting the password when it's non-empty?

Reply:
sure can do